src/js/pages/system/OverviewDetailTab.js

```diff
@@ -220,14 +250,29 @@ class OverviewDetailTab extends mixin(StoreMixin) {
    * @memberOf OverviewDetailTab
    */
   getMesosDetails() {
-    const mesosConfig = MesosStateStore.get("lastMesosState");
-    const mesosCluster = mesosConfig.cluster || this.getLoading();
-    const mesosLeaderInfo = mesosConfig.leader_info || this.getLoading();
-    const mesosVersion = mesosConfig.version || this.getLoading();
-    const mesosBuilt = mesosConfig.build_time || this.getLoading();
-    const mesosStarted = mesosConfig.start_time || this.getLoading();
-    const mesosElected = mesosConfig.elected_time || this.getLoading();
-    const mesosBuildUser = mesosConfig.build_user;
+    const {
+      masterInfo,
+      cluster,
+      version,
+      buildTime,
+      startTime,
+      electedTime,
+      buildUser
+    } = this.state;
+
+    const mesosMasterInfo = masterInfo || this.getLoading();
+    const mesosCluster = cluster || this.getLoading();
+    const mesosVersion = version || this.getLoading();
+    const mesosBuilt = buildTime
+      ? moment(buildTime * 1000).fromNow()
+      : this.getLoading();
+    const mesosStarted = startTime
+      ? moment(startTime * 1000).fromNow()
+      : this.getLoading();
+    const mesosElected = electedTime
```

Comment:
When I ran this the started and elected time were endlessly loading. The info.start_time was null and I see that the electedTime is not set somewhere. Is this intended?

![image](https://user-images.githubusercontent.com/180432/32291185-94c70660-bef9-11e7-9558-c688a8adbd98.png)

Reply:
Yep, as I pointed in the description, V1 API lacks elapsed_time and start_time, there's a ticket for that in mesos.